src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int constant = 1000;
```

Comment:
Change to the meaningful constant name
```suggestion
    private static final int <CHANGE_TO_PROPER_CONSTAN_NAME>= 1000;
```